test/loadTest/connections_stress_tests/connections_stress_tests.py

```diff
@@ -148,31 +148,35 @@ def __get_connection_info(self):
              can have a name like "contextBroker-1.4.3..." and the logic will still work
         :return tuple (established_conn, close_wait_conn)
         """
-        conn = rpyc.classic.connect(self.host)
-        connections_est = """def get_number_of_connections():
```

Comment:
where it is now -> I mean, "now" at master (i.e. previus to this PR proposed changes)